_posts/2021-03-30-influxdb-rails.html

```diff
@@ -0,0 +1,199 @@
+---
+image: fleur-dQf7RZhMOJU-unsplash.jpg 
+image_license: Photo by Fleur on <a href="https://unsplash.com/photos/dQf7RZhMOJU">Unsplash</a>
+title: Measure twice, instrument once with influxdb-rails!
+description: A blog post about the Ruby on Rails APM with influxdb-rails
+---
+
+<p>
+  Now that we learned how truly magnificent <code>ActiveSupport::Notifications</code> is in the previous post,
+  let's explore a RubyGem <a href="https://bruckmayer.net/">Chris</a>, others and me have built around this: influxdb-rails.
```

Comment:
I think it would be good to link to the gem's repository here, just in case people are getting curious.

```suggestion
  let's explore a RubyGem <a href="https://bruckmayer.net/">Chris</a>, others and me have built around this: [influxdb-rails](https://github.com/influxdata/influxdb-rails).
```

Reply:
I don't want to send people away in the lede...

Reply:
True